### ISSUE TITLE
vrv and goggle 2: electric boogaloo

### ISSUE THREAD

**Issue by ReiRchan** [samuelmaddock/metastream#169]
"Paste a website URL within the Metastream browser's address bar or into the chat to add it to the session."

I did that and nothing happens. I tried to do this several times each time in 2 days. The videos for vrv refuse to play! Also if you look up anything on goggle you can't go to any websites, since it refuses to load. It loads for one second and gives up, so unless you click on images you can't do anything on goggle. I got the extension on my browser and I can't do much ales to make it work. 

a 23 minutes video is what I'm trying to watch, but it seems to be a blank page that lasts for 5 seconds. This is after I post the direct URL into the search bar and put the add to the session button. I tried this several times and it does the same thing. 
![ex](https://user-images.githubusercontent.com/53590313/62342974-81699400-b4ae-11e9-9fb7-73e61d9b77ee.PNG)
session.

**Comment by samuelmaddock** [comment]
Do you have a premium subscription to VRV? There might be an ad which is 5 seconds long loading first.

**Comment by DizzyDean99** [comment]
I am also having this issue, I do pay for VRV, yet i still have that 5 second black box, before the link just closes. I tried doing a pause/play trick someone suggested in another chat, yet that failed to provide results too. I also tried just using VRV.co and selecting the video I wanted and that didn't change anything. I have also made sure im using the updated version of everything, and even made sure that Metastream remote has full access.

has any fixes been found? I also tried looking through the discord server but was quickly lost.
